src/main.rs

```diff
@@ -336,3 +379,30 @@ fn test_ignore_target() {
     );
     assert!(!entries.unwrap().is_empty());
 }
+
+#[test]
+fn test_remove_dependencies() {
+    let manifest = PathBuf::from(TOP_LEVEL).join("./integration-tests/multi-key-dep/Cargo.toml");
+    let stripped_manifest = remove_dependencies(
+        &std::fs::read_to_string(manifest).unwrap(),
+        &["cc".to_string(), "log-once".to_string(), "rand".to_string()],
+    )
+    .unwrap();
+    assert_eq!(
+        stripped_manifest,
+        r#"[package]
+name = "multi-key-dep"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+log = "0.4.14"
+
+[target.'cfg(unix)'.dependencies]
+
+[dev-dependencies]
+
+[build-dependencies]
```

Comment:
I used a direct string comparison wagering that this might be the easiest way to ensure removal worked as expected.